integration-tests/e2e-runtime-only-backend/lib.rs

```diff
@@ -156,7 +156,7 @@ pub mod flipper {
         }
 
         /// Just instantiate a contract using non-default runtime.
-        #[ink_e2e::test(backend = "runtime_only", runtime = ink_e2e::MinimalRuntime)]
+        #[ink_e2e::test(backend(runtime_only = ink_e2e::MinimalRuntime))]
```

Comment:
So, I would rather have 
```suggestion
        #[ink_e2e::test(backend(drink, runtime = ink_e2e::MinimalRuntime))]
```

Reply:
The easier here to avoid writing darling custom parsing code would be to use a `Drink{Runtime: Option<Syn::Path>}`
so that you can write `backend(drink())` for default and `backend(drink(foo))` for a `Some(foo)` value